.github/workflows/deploy.yml

```diff
@@ -89,6 +89,7 @@ jobs:
       - name: Install Dependencies
         run: |
           brew install cdrtools

```

Comment:
Do we still need cdrtools? I think this could possibly be orphaned dependency for mkisofs which is no longer used